.github/dependabot.yml

```diff
@@ -6,67 +6,24 @@ updates:
     interval: daily
   open-pull-requests-limit: 10
   ignore:
-  - dependency-name: mock
-    versions:
-    - "> 2.0.0"
-  - dependency-name: pillow
-    versions:
-    - ">= 7.a"
-    - "< 8"
   - dependency-name: publicsuffixlist
```

Comment:
Just curious, why are we still holding ``publicsuffixlist`` and ``qrcodegen`` back?

Reply:
They weren’t added because of Python 2 incompatibility.

- I linked qrcodegen’s commit log at the time, but I don’t remember why :D It might have been that most of the changes being made weren’t useful to us.
- publicsuffixlist upgrades make a lot of Dependabot noise since the public suffix list is always changing, and we don’t need the changes unless they affect domains we want to block.